services/common/src/redux/slices/verifiableCredentialsSlice.ts

```diff
@@ -4,6 +4,9 @@ import CustomAxios from "@mds/common/redux/customAxios";
 import { ENVIRONMENT, MINES_ACT_PERMITS_VC_LIST } from "@mds/common/constants";
 import * as API from "@mds/common/constants/API";
 import { RootState } from "@mds/common/redux/rootState";
+import { DataSourceItemType } from "antd/lib/auto-complete";
+import { debug } from "webpack";
+import { a } from "@mds/common/tests/mocks/dataMocks";
 
```

Comment:
What is 'a'? And is this + debug something that's no longer being used? I would guess it might cause issues with variables on `sort(a,b)`

Reply:
removed